README-legacy.md

```diff
@@ -0,0 +1,194 @@
+# Callback-based API
```

Comment:
Can you rename this file to `callback.md`?

Reply:
Done.